app/controllers/api/v1x0/rbac_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1x0
+    class RbacController < ApplicationController
```

Comment:
@syncrou This name is misleading would it be better to call it SeedController?

Reply:
Yes, I'll get that changed.

Reply:
@syncrou 
Since you would already be inside a request don't we just have to override this method
https://github.com/ManageIQ/catalog-api/blob/8541cbb7f360d5c3a81edd9420cd4a9a4b0826c3/lib/rbac/seed.rb#L10
and it would do all the seeding. When we use the rake task we have to create a fake request but in your case we just need to use an existing request and ignore the user file